apps/src/maze/wordsearch.js

```diff
@@ -1,213 +1,213 @@
-var color = require('../color');
-var _ = require('lodash');
-var cellId = require('./mazeUtils').cellId;
+import color from '../color';
+import _ from 'lodash';
+import { cellId } from './mazeUtils';
+import { SquareType } from './tiles';
+import { SVG_NS } from '../constants';
 
-var SquareType = require('./tiles').SquareType;
-
-var SVG_NS = require('../constants').SVG_NS;
-
-/**
- * Create a new WordSearch.
- */
-var WordSearch = module.exports = function (goal, map, drawTileFn) {
-  this.goal_ = goal;
-  this.visited_ = '';
-  this.map_ = map;
-};
-
-var ALL_CHARS = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L",
+const ALL_CHARS = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L",
```

Comment:
Because formerly, `START_CHAR` and `EMPTY_CHAR` were exposed along with `letterValue` and `randomLetter` in the tests, so I made them all class methods so they could still be accessed. But you're right that for consistency's sake this should be implemented in a parallel way.